eZ/Publish/Core/settings/search_engines/legacy/field_value_mappers.yml

```diff
@@ -0,0 +1,3 @@
+parameters:
+    ezpublish.search.common.field_value_mapper.date.class: eZ\Publish\Core\Search\Legacy\Content\FieldValueMapper\DateMapper
+    ezpublish.search.common.field_value_mapper.float.class: eZ\Publish\Core\Search\Legacy\Content\FieldValueMapper\FloatMapper
```

Comment:
@pspanja: it overrides default field value mappers for the legacy search engine. 
Default `DateMapper` returns date string - for legacy we need unix timestamp. 
Default `FloatMapper` returns `sprintf('%F', (float)$field->value)` - for legacy it returns just a `float` (the same in case of the *Elasticsearch*).